src/main/java/seedu/fast/logic/commands/MarkAppointmentCommand.java

```diff
@@ -0,0 +1,100 @@
+package seedu.fast.logic.commands;
+
+import static seedu.fast.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.fast.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.fast.commons.core.Messages;
+import seedu.fast.commons.core.index.Index;
+import seedu.fast.logic.commands.exceptions.CommandException;
+import seedu.fast.model.Model;
+import seedu.fast.model.person.Appointment;
+import seedu.fast.model.person.Person;
+
+public class MarkAppointmentCommand extends Command {
+
+    public static final String COMMAND_WORD = "done";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Marks an appointment with the person identified as completed"
+            + " by the index number used in the last person listing.\n\n"
+            + "Parameters (to add an appointment): \nINDEX (must be a positive integer), "
+            + "Examples: \n" + COMMAND_WORD + " 1 ";
+
+    public static final String MESSAGE_MARK_APPOINTMENT_SUCCESS = "Completed an appointment with "
+            + "%1$s %2$s %3$s %4$s";
+    public static final String MESSAGE_MARK_APPOINTMENT_FAILURE = "No Appointment exist!";
```

Comment:
MInor english nitpick, looks nicer if the appointment is not in caps since we never caps appointment in other messages anyway. (consistency)